src/Services/BitcoinService.cs

```diff
@@ -68,7 +69,7 @@ public async Task<(decimal, long)> GetWalletConfirmedBalance(Wallet wallet)
 
             var (_, explorerClient) = LightningHelper.GenerateNetwork();
 
-            var balance = await explorerClient.GetBalanceAsync(wallet.GetDerivationStrategy());
+            var balance = await explorerClient.Execute(x => x.GetBalanceAsync(wallet.GetDerivationStrategy(), default));
```

Comment:
what are these changes for?

Reply:
The explorer client doesn't have an interface so we can't mock it. We added the [Unmockable](uhttps://github.com/riezebosch/Unmockablerl) library so we can do that.
One of the disadvantages is that you have to change production code

Reply:
nice thanks. I'm not too worried about that

Reply:
Anyway this will change in #91